dotCMS/src/main/java/com/dotmarketing/portlets/workflows/business/WorkflowAPIImpl.java

```diff
@@ -111,8 +111,12 @@ public void registerBundleService () {
 		if(System.getProperty(WebKeys.OSGI_ENABLED)!=null){
 			// Register main service
 			BundleContext context = HostActivator.instance().getBundleContext();
-			Hashtable<String, String> props = new Hashtable<String, String>();
-			context.registerService(WorkflowAPIOsgiService.class.getName(), this, props);
+			if (null != context) {
+				Hashtable<String, String> props = new Hashtable<String, String>();
+				context.registerService(WorkflowAPIOsgiService.class.getName(), this, props);
+			} else {
+				Logger.debug(this, "Bundle Context is null, WorkflowAPIOsgiService has been not registered");

```

Comment:
@jdotcms please instead of a debug let's change it to error, if we are trying to register something with OSGI and we can not do it we want to be noisy about it and know the reason.